driver-core/src/test/java/com/datastax/driver/core/TableMetadataTest.java

```diff
@@ -474,4 +482,53 @@ public void should_not_mix_indexes_from_different_tables() {
         assertThat(table_cd.getIndexes().size()).isEqualTo(1);
         assertThat(table_cd.getIndexes().get(0).getName()).isEqualTo("test_d");
     }
+
+    /**
+     * Validates that the 'extensions' option is properly parsed when set.  This value is currently not
+     * modifiable via CQL so we fake out a row containing populated extensions by retrieving
+     * the current metadata and changing the extensions column.  The row is passed to the Options
+     * constructor and we validate the extensions map is matches what was set.
```

Comment:
altering the table after changing the extensions column sets it back to an empty map, so I explicitly call refreshSchema instead.